os/sdk-modifying-coreos.md

```diff
@@ -157,6 +165,8 @@ After `build_image` completes, it prints commands for converting the raw bin int
 
 Once you build an image you can launch it with KVM (instructions will print out after `image_to_vm.sh` runs).
 
+If you encounter errors with KVM, please ensure it is supported by your CPU and enabled in the BIOS.
```

Comment:
`please` is arguably contrary to the style guide.  :disappointed:  Maybe just `verify it is supported`?